uvtools/utils.py

```diff
@@ -99,3 +100,128 @@ def search_data(templates, pols, matched_pols=False, reverse_nesting=False, flat
         datapols = [item for sublist in datapols for item in sublist]
 
     return datafiles, datapols
+
+def FFT(data, axis):
+    """Convenient function for performing a FFT along an axis.
+
+    Parameters
+    ----------
+    data : np.ndarray
+       An array of data, assumed to not be ordered in the numpy FFT convention.
+       Typically the data_array of a UVData object.
+
+    axis : int
+        The axis to perform the FFT over.
+
+    Returns
+    -------
+    data_fft : np.ndarray
+        The Fourier transform of the data along the specified axis. The array
+        has the same shape as the original data, with the same ordering.
+    """
+
+    return fftshift(fft(fftshift(data, axis), axis=axis), axis)
+
+def get_fourier_freqs(times):
```

Comment:
The `get` prefix is normally used for methods in classes. `fourier_freqs` is fine here.